word_parser.rb

```diff
@@ -0,0 +1,52 @@
+module WordParser
+  def self.read_file(file_path)
+    File.open(file_path) { |f| f.read }
+  end
+
+  def self.to_arrayed_words(string)
+    string.gsub!(/\s\-\s/, ' ')
```

Comment:
It seems like it'd be nicer to deal with `' - '` later, when we're cleaning stuff. Maybe the `remove_empty_words` method could use a regex to look for words with no letters or numbers?